### DISCUSSION TITLE
Any issue with the latest version of Wireshark 4.2.3  ?

### DISCUSSION THREAD

**Discussion by vPackets** [siemens/edgeshark#50] [Q&A] [answered]
Hi guys !

I just upgraded my wireshark and my edgeshark containers:

$ ./edgesharkd.sh
[+] Running 6/6
 ✔ edgeshark 2 layers [⣿⣿]      0B/0B      Pulled                                                                                                                                                         8.1s
   ✔ ce715ac60835 Pull complete                                                                                                                                                                           4.0s
   ✔ e27fab22e5de Pull complete                                                                                                                                                                           4.1s
 ✔ gostwire 2 layers [⣿⣿]      0B/0B      Pulled                                                                                                                                                          7.1s
   ✔ 4abcf2066143 Already exists

I can reach the edgeshark GUI and see all container being linked together but when I try to sniff on one interface of a container, packetflix launches and I see wireshark being launched as well. As soon as the Wireshark GUI is rendered, both packetflix and Wireshark disapear from the dock and obviously crash .... 

Have you noticed the same behavior ?

Is there a way for me to troubleshoot that ?

Thanks a ton guys !

**Comment by vPackets** [comment, marked as answer]
Update: Upgraded to v0.10.7 [Latest](https://github.com/siemens/cshargextcap/releases/latest) and worked like a charm ....

I did use my already download version of csharextcap after I upgraded wireshark and it didnt work....

Good to know !

Thanks guys for the awesome work ... This is by FAR my favorite tool in conjunction with Containerlab

**Comment by thediveo** [comment]
Thanks for raising this anyway! I haven't seen this yet, but it absolutely helps to have this documented.
